runner.go

```diff
@@ -144,6 +144,70 @@ func (d *Runner) Run() error {
 	return err
 }
 
+// Run will validate that all edges in the graph point to existing vertices, and that there are
+// no dependency cycles. After validation, each vertex will be run, deterministically, in parallel
+// topological order. Difference between Run is it will continue execution despite vertex returning
+// an error (although will not execute it's dependencies).
```

Comment:
I would remove the parts that are duplicated from `Run`'s comment and just explain the difference.  (and proofread this comment for grammar)

Reply:
👍